### PR TITLE
spatial ui (part 1)

### PR DESCRIPTION
**Pull request by Alvareon**
This is the first step of new spatial UI integration:

- added new screens to ui_kit;
- ui_kit moved to another place;
- updated the PlatformManager load objects;
- added root prefab ViewSpatial for the spatial UI;
- refactoring of prefab structure;
- added Extensions scripts; 
- added View for Main and Profile screens;

### CODE REVIEW

**Review by fwild — CHANGES_REQUESTED** [review]
Looks excellent to me - only about organisation: shouldn't Utility.UiKit be inside the MirageXR folder?

File: Assets/MirageXR/Player/Scripts/Spatial/MainScreenSpatialView.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+using Utility.UiKit.Runtime.Extensions;
+
+namespace MirageXR
+{
+    public class MainScreenSpatialView : View
+    {
+        [Header("Buttons")]
+        [SerializeField] private Button _buttonSidebarCollapse;
+        [SerializeField] private Button _buttonSidebarExpand;
+        [SerializeField] private Button _button1;
```
Comment (cawke) [inline Assets/MirageXR/Player/Scripts/Spatial/MainScreenSpatialView.cs]: change the name

File: Assets/MirageXR/Design/UI_Elements/PNG/Icons/Spatial/Icon_ShowPass.png.meta [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  streamingMipmaps: 0
+  streamingMipmapsPriority: 0
+  vTOnly: 0
+  ignoreMipmapLimit: 0
+  grayScaleToAlpha: 0
+  generateCubemap: 6
+  cubemapConvolution: 0
+  seamlessCubemap: 0
+  textureFormat: 1
+  maxTextureSize: 2048
```
Comment (cawke) [inline Assets/MirageXR/Design/UI_Elements/PNG/Icons/Spatial/Icon_ShowPass.png.meta]: change max resolution to 256

**Review by cawke — CHANGES_REQUESTED** [review]

**Comment by Alvareon** [comment]
> Looks excellent to me - only about organisation: shouldn't Utility.UiKit be inside the MirageXR folder?

UiKit moved into the MirageXR folder

File: Assets/MirageXR/Player/Scripts/Spatial/MainScreenSpatialView.cs (hunk 1 above)
Comment (Alvareon) [reply Assets/MirageXR/Player/Scripts/Spatial/MainScreenSpatialView.cs]: fixed

File: Assets/MirageXR/Design/UI_Elements/PNG/Icons/Spatial/Icon_ShowPass.png.meta (hunk 2 above)
Comment (Alvareon) [reply Assets/MirageXR/Design/UI_Elements/PNG/Icons/Spatial/Icon_ShowPass.png.meta]: fixed

**Review by fwild — APPROVED** [review]
please merge :)

**Pull request merged**